src/sentry/integrations/msteams/webhook.py

```diff
@@ -362,14 +365,15 @@ def handle_action_submitted(self, request: Request):
         team_id = integration.external_id
         client = MsTeamsClient(integration)
 
-        try:
-            group = Group.objects.select_related("project__organization").get(
-                project__in=Project.objects.filter(
-                    organization__in=integration.organizations.all()
-                ),
-                id=group_id,
+        group = Group.objects.select_related("project__organization").filter(id=group_id).first()
+        if group:
+            integration = integration_service.get_organization_integration(
```

Comment:
@corps looks like this change broke msteams linking :(

Reply:
@corps this is fetching the org integration instead of the integration. Working on a fix now.